contributor-guide/modules/ROOT/pages/getting-involved.adoc

```diff
@@ -1,4 +1,58 @@
 = Getting Involved
 :navtitle: Getting Involved
 
-*Content under construction.*
\ No newline at end of file
+There are many ways to get involved with Boost, up to and including submitting your own library. In roughly ascending order of time and work commitment, here are the ways you can contribute to Boost:
+
+== 1. Join the Boost Users Mailing List
```

Comment:
One problem with this is this page is in the "Contributor Guide" component. Usually, users join the user's mailing list and contributors join the mailing list for developers.

Reply:
OK, corrected to prioritize the develoers list.

Reply:
> here are the ways you can contribute to Boost:
>
> == 1. Join the Boost Users Mailing List

This sounds a little bit weird. It's true that you should join the ML if you want to get involved, but joining the ML is not contributing per se.

Reply:
Bit semantic but true - so changed this to "Participate in the Boost Developers Mailing List"

Reply:
> semantic

To be clear, we're not calling the same ML a different name to fit the text better. 

They are literally two different mailing lists with different addresses and different rules.